semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/linking/calllinker/CallLinker.scala

```diff
@@ -61,13 +67,19 @@ class CallLinker(cpg: Cpg) extends CpgPass(cpg) {
                 ._refOut
                 .onlyChecked
 
-              receiverTypeDecl._bindsOut.asScala.collectFirst {
+              val res = receiverTypeDecl._bindsOut.asScala.collectFirst {
                 case binding: nodes.Binding if binding.name == call.name && binding.signature == call.signature =>
                   binding._refOut.onlyChecked.asInstanceOf[nodes.Method]
               }
+              if (res == None) {
+                logger.debug(
```

Comment:
I would never log in this case.

Reply:
*shrugs*

The old log message was presumably useful (otherwise it wouldn't have been there, right?), just too loud. Hence make info->debug.

Reply:
This case went from being considered a problem to now being considered valid input. So why should we log it?